### PR TITLE
Updating Hex Stickers in README

### LINKED ISSUE #28: update thumbs folder
I just noticed we have `thumbs` folder:
https://github.com/insightsengineering/hex-stickers/tree/main/thumbs

We probably need to update this too.

@pawelru can we ask the designer to create the thumbnail files?

### PR DESCRIPTION
**Pull request by donyunardi**
Fix #28 

`README.Rmd` has a section to update the content in `thumbs` folder every time a new content is added in PNG and SVG.
Re-running the README.Rmd file to update the Hex Stickets (wall and details).

Here's the new look of the hexwall:
image

Hex details also updated.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,6 +21,12 @@ the thumbnail(s) for your logo(s).
 
 ## Hex sticker wall
 
+    ## Warning: `invoke()` was deprecated in purrr 1.0.0.
+    ## ℹ Please use `exec()` instead.
+    ## This warning is displayed once every 8 hours.
+    ## Call `lifecycle::last_lifecycle_warnings()` to see where this warning was
+    ## generated.
```
Comment (pawelru) [inline README.md:28]: Hi @donyunardi. This is good but we got following warning. Would you be able to modify `.Rmd` file accordingly and re-generate `.md` afterwards?

**Comment by donyunardi** [comment]
Thanks for the feedback @pawelru !
I updated `invoke` to `exec` and regenerate `.md`

**Review by pawelru — APPROVED** [review]
LGTM 👍

**Comment by averissimo** [comment]
@donyunardi this fails the github action as the number of images has increased and this exausts the stack of the image processing

This PR should solve it: https://github.com/insightsengineering/hex-stickers/pull/30

**Pull request merged**